ansible/roles-infra/infra-osp-template-create/tasks/create_osp_template.yml

```diff
@@ -1,69 +1,98 @@
+- name: Check for running Heat stacks
+  command: >-
+    openstack stack list -f json
+  environment:
+    OS_AUTH_URL: "{{ osp_auth_url }}"
+    OS_USERNAME: "{{ osp_auth_username | default(osp_auth_username_member) }}"
+    OS_PASSWORD: "{{ osp_auth_password | default(osp_auth_password_member) }}"
+    OS_PROJECT_NAME: "{{ osp_project_name }}"
+    OS_PROJECT_DOMAIN_ID: "{{ osp_auth_project_domain }}"
+    OS_USER_DOMAIN_NAME: "{{ osp_auth_user_domain }}"
+  register: r_stack_list
+  until: 
+  - r_stack_list.stdout | from_json | json_query(create_stacks) | length <= max_create_in_progress
+  - r_stack_list.stdout | from_json | json_query(update_stacks) | length <= max_update_in_progress
+  retries: 10
+  delay: "{{ 300 | random(start=120) }}"
+  vars:
+    create_stacks: >-
+      [?"Stack Status"=='CREATE_IN_PROGRESS']
+    update_stacks: >-
+      [?"Stack Status"=='UPDATE_IN_PROGRESS']
+
+- debug:
```

Comment:
Tasks should have names.

Reply:
my understanding is that we are not going to support stack updating.  can we filter the query using `openstack stack list --property stack_status=CREATE_IN_PROGRESS -f json` and remove all the updating stuff?